src/NewTools-Playground/StPlaygroundInteractionModel.class.st

```diff
@@ -62,6 +62,11 @@ StPlaygroundInteractionModel >> bindings [
 	^ joinedBindings
 ]
 
+{ #category : #'interactive error protocol' }
+StPlaygroundInteractionModel >> correctFrom: anInteger to: anInteger2 with: aString [ 
```

Comment:
can't we have better names for those variables?

Reply:
Done!